my-app/src/public-components/text-link/text-link.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import classes from './text-link.css';
+import './text-link.css';
 
 
 
-export const Textlink = (props) => {
+export const Textlink = ({ path, onClick, name }) => {
 
   return (  
     <span className="text_link">
-        <Link to={props.path}>{props.name}</Link>
+        <Link to={path} onClick={onClick}>{name}</Link>
```

Comment:
Link component must be link component only, if you want use onClick, then create separate component text-button component